release/release.py

```diff
@@ -127,14 +127,17 @@ def command_stage_docs():
     if not os.path.exists(kafka_site_repo_path) or not os.path.exists(os.path.join(kafka_site_repo_path, "powered-by.html")):
         fail("{kafka_site_repo_path} doesn't exist or does not appear to be the kafka-site repository")
 
-    jdk21_env = get_jdk(21)
+    jdk25_env = get_jdk(25)
```

Comment:
https://github.com/apache/kafka/blob/92d9dc19b22529895c430bf2551b551cbfbcaad1/release/templates.py#L28C5-L28C42

@apalan60 could you please update the template too?